app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -67,6 +67,15 @@ define([
       return ! account.has('profileImageUrl');
     },
 
+    logAccountImageChange: function (account) {
+      // if the user already has an image set, then report a change event
+      if (account.has('profileImageId')) {
```

Comment:
Let's use `profileImageUrl` instead since there's precedent for using that attribute.